src/Sylius/Bundle/UserBundle/Form/Type/CustomerSimpleRegistrationType.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\UserBundle\Form\Type;
+
+use Sylius\Bundle\ResourceBundle\Form\Type\AbstractResourceType;
+use Sylius\Bundle\UserBundle\Form\EventListener\CustomerRegistrationFormListener;
+use Sylius\Bundle\UserBundle\Form\EventListener\UserRegistrationFormListener;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+/**
+ * @author Michał Marcinkowski <michal.marcinkowski@lakion.com>
+ */
+class CustomerSimpleRegistrationType extends AbstractResourceType
+{
+    /**
+     * @var RepositoryInterface
+     */
+    private $customerRepository;
+
+    /**
+     * @param string              $dataClass
+     * @param array               $validationGroups
+     * @param RepositoryInterface $customerRepository
+     */
+    public function __construct($dataClass, array $validationGroups = array(), RepositoryInterface $customerRepository)
```

Comment:
Do not add default value here. Also, convention is that required arguments are always first. :)